README.md

```diff
@@ -62,24 +62,23 @@ Issues labeled [help wanted](https://github.com/photoprism/photoprism/labels/hel
 [easy](https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy) can be
 good (first) contributions. 
 
-Follow us on [Twitter][twitter] to receive regular project updates and discuss development 
-related topics. Don't be afraid to ask stupid questions.
+Follow us on [Twitter][twitter] to receive regular project updates and discuss development related topics. Don't be afraid to ask stupid questions.
 
 ## Funding ##
 
 This software is about freedom and privacy but not necessarily about free beer. We feel like it
-would be a mistake to state there will be no costs, because clearly we have huge expenses, your server hardware
+would be a mistake to state there will be no costs because clearly, we have huge expenses, your server hardware
 will have a price tag, and then maybe you'd like to have some extra features that need to be developed.
 
 You're welcome to support us via [GitHub Sponsors](https://github.com/sponsors/photoprism), 
-especially if you have feature requests, or need help with using our software.
+especially if you have feature requests or need help with using our software.
 In addition, you can find us on [Patreon](https://www.patreon.com/photoprism)
 and [PayPal](https://www.paypal.me/photoprism). 
 
 Your continuous support helps...
 
-* to pay for operating expenses and external services like satellite maps
-* developing new features, and keeping them free for everyone 🌈
+* To pay for operating expenses and external services like satellite maps
```

Comment:
Why uppercase?

Reply:
To match the rest of the bullet points in the document.

Reply:
Huh, let me revert this. Not sure how I missed the next bullet isn't either.